packages/code-du-travail-frontend/src/layout/Header/index.js

```diff
@@ -123,6 +118,7 @@ const LogoLink = styled.a`
 
 const Marianne = styled.img`
   width: 7rem;
+  height: 7rem;
```

Comment:
fix le logo croppé en bas

Reply:
original image is 153px x 150px so fixing 7rem x 7rem made it blury. why not use `height: default` ?

Reply:
It is always blury because the text is too small ^^ I tried lots of variation but none work, neither height 100%, or default or auto, or some align-items stretch on the parent (does the same as fixing width and height to the same) or flex-shrink 0 & so. Even reducing image width keeps the bottom to be cropped for some reasons I don't get.
He is a gif, I don't see any difference in blur to be honest
![no-blur](https://user-images.githubusercontent.com/705453/79893399-4de7e480-8404-11ea-8b01-cefa68cae7c7.gif)